packages/vsce/__tests__/__e2e__/00_CreateNewGlobalTeamConfiguration.test.ts

```diff
@@ -0,0 +1,110 @@
+
+describe("Create New Global Team Configuration File Scenario", () => {
+  let cicsTree: DefaultTreeSection;
+  let quickPick: InputBox;
+  let dialog: ModalDialog;
+  let element: WebElement;
+  let plusIcon: ViewPanelAction;
+  let editorView: EditorView;
+
+  before(async function () {
+    this.timeout(15000);
+    // open the cics extension view
+    const view = await(await new ActivityBar().getViewControl("Zowe Explorer"))?.openView();
+
+    //handle notifications if config file is missing
+    const notifications = await new Workbench().getNotifications();
+    if (notifications.length > 0) {
+      await notifications[0].click();
+      const actions = await notifications[0].getActions();
+      await actions[0].click();
+
+      //select qp option
+      quickPick = await InputBox.create();
+      await quickPick.selectQuickPick(0);
+    }
+
+    cicsTree = await view.getContent().getSection("cics");
+    await cicsTree.click();
+  });
+
+  it("CICS Section title Check", async () => {
+    // title check for cics section
+    const title = await cicsTree.getTitle();
+    expect(title).equals("cics");
+  }).timeout(5000);
+
+  it("+ (plus) button clickable test", async () => {
+    // selecting cics view and clicking + icon
+    await cicsTree.click();
+    await cicsTree.expand();
+    plusIcon = await cicsTree.getAction(`Create a CICS Profile`);
+    expect(plusIcon).exist;
+    cicsTree.takeScreenshot();
+    await plusIcon.click();
+  }).timeout(5000);
+
+  it("Quick Pick Option test", async () => {
+    // Find quickpick
+    quickPick = await InputBox.create();
+    const qpItems = await quickPick.getQuickPicks();
+
+    const label1 = await qpItems[0].getLabel();
+    expect(label1).contains("＋ Create a New Team Configuration File");
+
+    if (qpItems.length > 1) {
+      const label2 = await qpItems[1].getLabel();
+      expect(label2).contains("Edit Team Configuration File");
+    }
+
+    const placeholder = await quickPick.getPlaceHolder();
+    expect(placeholder).equals('Choose "Create new..." to define or select a profile to add to the CICS tree');
+    cicsTree.takeScreenshot();
+  }).timeout(5000);
+
+  // now we can check what the dialog says
+  it("Create New Configuration File Test", async () => {
+    quickPick = await InputBox.create();
+    await quickPick.selectQuickPick(0);
+    dialog = new ModalDialog();
+    await dialog.click();
+
+    element = dialog.getEnclosingElement();
+    await element.click();
+
+    const buttons = await dialog.getButtons();
+
+    // there should be 2 buttons
+    expect(buttons.length).equals(2);
+
+    // or we can directly push a button by title
+    await dialog.pushButton(`Create New`);
+
+    // Find open editors
+    editorView = new EditorView();
+    const titles = await editorView.getOpenEditorTitles();
+
+    // Check zowe.config.json was opened - could check content here
+    expect(titles.some((title) => title.startsWith("zowe.config.json"))).is.true;
```

Comment:
This is good. Wonder if we can check more than just the title - perhaps some of the filepath to know we've got the corrct config file (global/project)